### PR TITLE
Handle ephemeral error status code

### PR DESCRIPTION
**Pull request by joshblum**
depends on https://github.com/keybase/keybase/pull/3264. Handles new server errors for ephemeral key failure conditions

### CODE REVIEW

File: go/ephemeral/errors.go [hunk 1]
```diff
@@ -52,15 +54,30 @@ func newEKCorruptedErr(ctx context.Context, g *libkb.GlobalContext, boxType EKTy
 	return newEphemeralKeyError(debugMsg, "")
 }
 
-func newEphemeralKeyError(debugMsg, humanMsg string) EphemeralKeyError {
+func humanMsgWithPrefix(humanMsg string) string {
 	if humanMsg == "" {
-		humanMsg = defaultHumanErr
+		humanMsg = DefaultHumanErrMsg
+	} else if !strings.Contains(humanMsg, DefaultHumanErrMsg) {
```
Comment (zapu) [inline go/ephemeral/errors.go]: I'm not sure what's going on here but I assume it's good

Comment (joshblum) [reply go/ephemeral/errors.go]: just a sanity check so we don't append the default message twice

**Comment by joshblum** [comment]
screen shot 2018-12-14 at 6 00 30 pm

**Comment by joshblum** [comment]
drops `Unable to decrypt chat message:` for ephemeral errors only

screen shot 2018-12-17 at 10 10 17 am

**Pull request merged**